Dockerfile

```diff
@@ -1,21 +1,28 @@
-FROM golang:1.10-alpine
+FROM golang:1.11-alpine
 
 ENV CGO_ENABLED=0
 
 RUN apk add --no-cache \
     git
 
-RUN go get -v \
-    github.com/stretchr/testify \
-    github.com/kisielk/errcheck
+RUN go get -v github.com/kisielk/errcheck/
 
-WORKDIR /go/src/github.com/section-io/beacon
-COPY *.go ./
+# WORKDIR /go/src/github.com/section-io/beacon
+WORKDIR /src
+COPY *.go go.mod go.sum ./
```

Comment:
```
COPY go.mod go.sum ./
RUN go mod download
COPY *.go ./
```
This will avoid re-downloading the modules everytime any change is made only to Go source code.

Reply:
Fixed, thanks for the tip!